dump/plugins/fdb.py

```diff
@@ -0,0 +1,103 @@
+from dump.match_infra import MatchRequest
+from dump.helper import create_template_dict
+from .executor import Executor
+
+
+class Fdb(Executor):
+    """
+    Debug Dump Plugin for FDB Module
+    """
+    ARG_NAME = "Vlan:fdb_entry"
+
+    def __init__(self, match_engine=None):
+        super().__init__(match_engine)
+
+    def get_all_args(self, ns=""):
+        req = MatchRequest(db="STATE_DB", table="FDB_TABLE", key_pattern="*", ns=ns)
+        ret = self.match_engine.fetch(req)
+        fdb_entries = ret["keys"]
+        return [key.split("|")[-1] for key in fdb_entries]
+
+    def execute(self, params):
+        self.ret_temp = create_template_dict(dbs=["APPL_DB", "ASIC_DB", "STATE_DB"])
+        fdb_entry = params[Fdb.ARG_NAME]
+        self.ns = params["namespace"]
+        self.init_fdb_appl_info(fdb_entry)
+        self.init_asic_fdb_info(fdb_entry)
+        self.init_state_fdb_info(fdb_entry)
+        return self.ret_temp
+
+    def add_to_ret_template(self, table, db, keys, err):
```

Comment:
This can be removed. the method has been added to the parent class. https://github.com/Azure/sonic-utilities/commit/4e132c1c86c515076815a64cf22aaf8e0101de84